cocos/network/CCDownloader-curl.cpp

```diff
@@ -845,7 +847,7 @@ namespace cocos2d { namespace network {
             DLLOG("    DownloaderCURL: finish Task: Id(%d)", coTask.serialId);
         }
         
-        if (_impl->stoped())
+        if (!_breakScheduleFunc && _impl->stoped())
```

Comment:
Current object destruct is raised by a function inside this object, so I think before the function exist, the object won't be really released.